chall01/xtang.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/python3
+
+import sys
+
+def errorMessage():
+	print(f"usage: {sys.argv[0]} <a-zA-Z string>")
+	exit()
+
+morse_dict = {
+"a": ".-",
+"b": "-...",
+"c": "-.-.",
+"d": "-..",
+"e": ".",
+"f": "..-.",
+"g": "--.",
+"h": "....",
+"i": "..",
+"j": ".---",
+"k": "-.-",
+"l": ".-..",
+"m": "--",
+"n": "-.",
+"o": "---",
+"p": ".--.",
+"q": "--.-",
+"r": ".-.",
+"s": "...",
+"t": "-",
+"u": "..-",
+"v": "...-",
+"w": ".--",
+"x": "-..-",
+"y": "-.--",
+"z": "--.."
+}
+
+def check_chars(word, morse_dict):
+	for x in word:
+		if x.isalpha() == False:
+			if x.isspace() == False:
+				errorMessage()
+		else:
+			if x not in morse_dict:
+				errorMessage()
```

Comment:
isalpha() can be used in a string or a char 

you also know that your dict keys is all the a-z lowercase.

I would recommend changing all of this by

```python
if not word.replace(" ", "").isalpha():
  errorMessage()
```